omod/src/main/java/org/openmrs/web/controller/OptionsFormController.java

```diff
@@ -182,13 +182,7 @@ protected ModelAndView onSubmit(HttpServletRequest request, HttpServletResponse
 				catch (APIException e) {
 					errors.rejectValue("oldPassword", "error.password.match");
 				}
-			} else {
-				// if they left the old password blank but filled in new
-				// password
-				if (!"".equals(opts.getNewPassword())) {
```

Comment:
What is the reason behind removing these lines?

Reply:
I expected those lines to be the second password field not needed

Reply:
Did you confirm?

Reply:
No I did not.

Reply:
As for this I also don't see it's effectiveness to remove one password field I'll also have to review my code changes made